voc/python/methods.py

```diff
@@ -1093,8 +1093,9 @@ def visitor_setup(self):
     def visitor_teardown(self):
         if len(self.opcodes) == 0 or not isinstance(self.opcodes[-1], JavaOpcodes.ATHROW):
             self.add_opcodes(
-                java.New('org/python/exceptions/StopIteration'),
-                java.Init('org/python/exceptions/StopIteration'),
+                JavaOpcodes.GETSTATIC('org/python/exceptions/StopIteration', 'STOPITERATION', 'Lorg/python/exceptions/StopIteration;'),
+                #java.New('org/python/exceptions/StopIteration'),
+                #java.Init('org/python/exceptions/StopIteration'),
```

Comment:
No need to keep these as commented code; just delete them.

Reply:
Done!